<launch>
  <param name="calibration_path" value="$(find rpl)/config/camera.xml" />
  <param name="file_path" value="$(find rpl)/config/source_position/"/>

  <node name="camera" pkg="rpl" type="camera.py" output="screen"/>
  <node name="ar" pkg="rpl" type="ar_detector.py" output="screen"/>
  <node name="ar_image_view" pkg="image_view" type="image_view" output="screen">
    <remap from="image" to="/AR/camera_image" />
  </node>
  <node name="source_position_generator" pkg="rpl" type="source_position_generator_minimini2_global.py" output="screen"/>
</launch>
